src/utils/cloudinary.js

```diff
@@ -40,8 +40,8 @@ const handleCldError = (that, options) => {
           goodSrcs.forEach(s => {
             s.try = true;
           });
+          that.videojs.autoplay(that.videojs.autoplay() || that.playWasCalled);
```

Comment:
What if autoplay = false, player.play was called and then player.pause was called and then player.source was called? it will set autoplay to true, which is not what we want. no?

Reply:
this is only relevant to the state before the video was started as it takes care of fallback.
I can't really see why someone would make all these calls.

Reply:
Take an interface similar to youtube as an example, you could play->pause->switch video, in such case you'll get autoplay even if autoplay is set to false. this might not be the wanted behavior.